doc-content/shared-kie-docs/src/main/asciidoc/Install/testing-clustering-environment-proc.adoc

```diff
@@ -0,0 +1,122 @@
+[id='testing-clustering-environment-proc_{context}']
+= Testing that your infrastructure meets the minimum hardware and performance requirements for a viable high availability (HA) on-premises environment.
+
+When creating a production-ready high availability (HA) on-premises infrastructure for {CENTRAL}, you must ensure it meets the minimum hardware and performance requirements for a viable HA environment, and to mitigate performance related issues. This section describes the environment elements, an example environment, and how to test the performance.
+
+There are four main components; {CENTRAL}, the message system (AMQ), the indexing server ({DATAGRID}), and a shared file system (NFS/GlusterFS/Ceph).
+
+.Prerequisites
+
+* A network environment of at least 3 nodes is configured with the following layout:
++
+Node 1: {CENTRAL}
++
+Node 2: {CENTRAL}
++
+Node 3: AMQ, {DATAGRID}, and NFS
+
+.Procedure
+
+. To test the network speed, do the following tasks:
+.. In the command terminal of each server node, install `iPerf3`:
++
+----
+$ dnf install iperf3
+----
+.. In the command terminal of the NFS server node (server node 3), start `iPerf3` in server mode:
++
+----
+$ iperf3 -s
+----
+.. In the command terminal of each {CENTRAL} server node, start `iPerf3` in client mode with the NFS server node set as the host:
++
+----
+$ iperf3 -c <NFS_SERVER_IP>
+----
+.. Note the results from each server node.
```

Comment:
For all three steps, I'd consolidate these two substeps or really just nix the step of noting the results, as follows.

```
.. Compare the results from each server node with the following example of minimum values:
```

Side note, the usage is typically "compare with" instead of "compare against", per Chicago, MW, etc.

Reply:
Agreed and done.